doc/api/debugger.md

```diff
@@ -145,6 +145,7 @@ debug>
 ### Information
 
 * `backtrace`, `bt` - Print backtrace of current execution frame
+* `frame(3)`, `f(3)` - Select stack frame #3 visible in the backtrace
```

Comment:
Yea I just meant `n` in the generic sense. Hard coding `3` is misleading, as is the `5` in the `list` example.